personio-personnel-data-api-oa3.yaml

```diff
@@ -2284,6 +2284,65 @@ paths:
                       code: 0
                       message: 'Something went wrong'
 
+  "/company/absence-periods/{id}":
+    parameters:
+      - $ref: "#/components/parameters/XPersonioPartnerID"
+      - $ref: "#/components/parameters/XPersonioAppID"
+    delete:
+      security:
+        - BearerAuth: []
+      tags:
+        - Absences
+      description: Deletes absence period data for absence types **tracked in hours**.
+      parameters:
+        - name: id
+          in: path
+          description: ID of the absence period to delete
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: 61fe126b-a7b3-449a-b5ee-3865a6fcc546
+      responses:
+        "200":
+          description: Success response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DeletedAbsenceResponse"
+              examples:
+                response:
+                  value:
+                    success: true
+                    data:
+                      message: The absence period was deleted.
+        "404":
+          description: Not found response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorAbsenceResponse"
```

Comment:
suggestion(blocking): The schema of ErrorAbsenceResponse and ErrorDeletingAbsenceInvalidDataResponse seem to be identical. Could you pleas reuse the error response schema?

Reply:
but the former is a 404 not found error and the latter is a 400 bad request error, that's why I think they're complementary and not identical, but I'm thinking about renaming ErrorAbsenceResponse to ErrorAbsenceNotFoundResponse would be useful here to be more clear what these errors represent, what do you think?

Reply:
Ahh sorry true I got your point, changing it 👍

Reply:
I just got confused as I saw the same schema duplicated for different errors in absences like `ErrorInsertingAbsenceResponse` and `ErrorCreateAbsenceResponse`, so I thought we're adopting this style 👍

Reply:
Fair point, we should think about consolidating the error response schema. But this is out of scope of this patch 😄